lib/btc/BtcRpc.js

```diff
@@ -71,7 +74,7 @@ class BtcRpc {
     return tx;
   }
 
-  async unlockAndSendToAddressMany({ payToArray, passphrase, time = 10800, maxValue = 10*1e8, maxOutputs = 1 }) {
+  async unlockAndSendToAddressMany({ account, payToArray, passphrase, time = 10800, maxValue = 10*1e8, maxOutputs = 1 }) {
```

Comment:
missing conf target

Reply:
other stuff as well, but most relevant would probably be conf target, minconf would also be nice

Reply:
made an issue for this: https://github.com/bitpay/crypto-rpc/issues/70